client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -53,7 +53,6 @@ function ReaderSubscriptionListItem( {
 	const streamUrl = getStreamUrl( feedId, siteId );
 	const feedUrl = url || getFeedUrl( { feed, site } );
 	const siteUrl = getSiteUrl( { feed, site } );
-	const isFollowing = ( site && site.is_following ) || ( feed && feed.is_following );
```

Comment:
goooooood catch. yikes.

maybe we should poke around and see if we're using site or feed.is_following anywhere else...